### ISSUE TITLE
Bug - [SelectColumn] -  label intended to increase clickable area has no effect in PF6

### ISSUE THREAD

**Issue by dgdavid** [patternfly/patternfly#7673]
**Describe the problem**

It looks like the label wrapping the input in `SelectColumn`, which was originally added in PatternFly 4 to increase the clickable area, no longer seems to have any effect in PatternFly 6:

https://github.com/patternfly/patternfly-react/blob/87ddd2d8cf02bacbf69c8950daff0137a9fc5c74/packages/react-table/src/components/Table/SelectColumn.tsx#L35

**How do you reproduce the problem?**

Visit https://www.patternfly.org/components/table#selectable-radio-input and try clicking just outside but near the radio input. It doesn’t get selected.

Then compare this behavior with the archived PatternFly 4 version at https://v4-archive.patternfly.org/v4/components/table#composable-selectable-radio-input. Here, clicking in the surrounding "column area" does select the radio input, and the cursor style differs as well.

**Expected behavior**

Not entirely sure what the behavior should be, but I personally lean toward increasing the actual input size as somehow mentioned in 
https://github.com/patternfly/patternfly/pull/4546#issuecomment-984767013

In my view, making native elements more prominent feels more appropiated than using a *transparent label* to expand the clickable area, which could be unexpected regarding to native behavior.

**Is this issue blocking you?**

Not at all. Just noticed this while reviewing the code and was curious about the intention behind such a  wrapping label.

**Any other information?**

Such a label was added at https://github.com/patternfly/patternfly-react/pull/6754 in patternfly-react package after previously added at https://github.com/patternfly/patternfly/pull/4546 in patternfly package.

---

**Jira Issue:** [PF-2398](https://redhat.atlassian.net/browse/PF-2398)

**Comment by thatblindgeye** [comment]
@mcoker @andrew-ronaldson could either of you provide input as to any preferred spacing/padding to apply for this?

**Comment by mcoker** [comment]
@andrew-ronaldson @lboehling what do you think about increasing the clickable area of the [standalone checkbox/radio components](https://www.patternfly.org/components/forms/checkbox#standalone-input)? The native target size for those elements is really small.

That would apply to all standalone radios/checkboxes. It wouldn't change the layout/look or anything, we would just expand the clickable area. The red border here shows what that might look like

<img width="188" height="80" alt="Image" src="https://github.com/user-attachments/assets/34ac0e70-55a5-4c90-99c8-43d9610f4b97" />

**Comment by andrew-ronaldson** [comment]
I think it's fine to change. Make it so!

Please

**Comment by mcoker** [comment]
To anyone who wants to work on this:
* We can make this change to the [standalone radio](https://www.patternfly.org/components/forms/radio/#standalone-input) and [standalone checkbox ](https://www.patternfly.org/components/forms/checkbox/#standalone-input), and the table will inherit this change.
* We can use a pseudo element on the outer label to increase the clickable area.
* The layout/shape of the radio and checkbox should not change.

In order to see the change in react, the `<Table>` will need to be updated to use the standalone radio/check. I created an issue for that here - https://github.com/patternfly/patternfly-react/issues/12018

**Comment by tarunvashishth** [comment]
I’ve opened a PR that addresses this issue: #8083
